build.gradle

```diff
@@ -136,27 +161,23 @@ task lobbyServer(type: Zip, group: 'release', dependsOn: ':shadowJar') {
 	from(configurations.testCompile.files { dep -> dep.name == 'derby' }) {
 		into('lib')
 	}
-	from("build/libs/triplea-${project.version}-all.jar") {
+	from(renameShadowJar.output) {
 		into('bin')
-		rename "triplea-${project.version}-all.jar", 'triplea.jar'
 	}
 }
 
 task generateZipReleases(group: 'release', dependsOn: [allPlatform, bots, lobbyServer]) {}
 
-task shadowJarCopy(type: Copy) {
-	from "build/libs/triplea-${project.version}-all.jar"
-	into 'build/libs'
-	rename { String fileName ->
-		fileName.replace("triplea-${project.version}-all.jar", "triplea.jar")
-	}
-}
-
-
-import com.install4j.gradle.Install4jTask
 apply plugin: 'install4j'
 
-task generateInstallers(type: Install4jTask, dependsOn: [ ':shadowJar', shadowJarCopy ],  group: 'release') {
+task copyJRE(type: Copy, dependsOn: [updateAssets]) {
+	from "${assetsDirectory}/install4j/windows-x86-1.8.0_66.tar.gz"
+	from "${assetsDirectory}/install4j/macosx-amd64-1.8.0_66.tar.gz"
+	from "${assetsDirectory}/install4j/windows-amd64-1.8.0_66.tar.gz"
+	into "${System.properties['user.home']}/.install4j6/jres"
```

Comment:
Again, pretty sure the path needs to be "~/install4j6", note "~/.install4j6", please verify.